developer_setup.md

```diff
@@ -19,9 +19,8 @@
   sudo dnf -y install postgresql-devel postgresql-server # PostgreSQL Database server and to build 'pg' Gem
   sudo dnf -y install bzip2 libffi-devel readline-devel  # For rbenv install 2.2.0 (might not be needed with other Ruby setups)
   sudo dnf -y install libxml2-devel libxslt-devel patch  # For Nokogiri Gem
-  sudo dnf -y install gcc-c++                            # For event-machine Gem
   sudo dnf -y install sqlite-devel                       # For sqlite3 Gem
-  sudo dnf -y install nodejs npm                         # For ExecJS Gem and bower
+  sudo dnf -y install nodejs                             # For ExecJS Gem and bower
```

Comment:
The new `nodejs` contains line
```
Obsoletes: npm < 0:3.5.4-6
Provides: npm = %{epoch}:%{npm_version
```
so the `dnf install npm` should continue to work anyway.

While I applaud to the clean-up, I am concerned that the missing npm may confuse centos7 users. What do you think?